modules/issue_tracker/php/issuerowprovisioner.class.inc

```diff
@@ -50,7 +50,7 @@ class IssueRowProvisioner extends \LORIS\Data\Provisioners\DBRowProvisioner
                     i.assignee,
                     i.status,
                     i.priority,
-                    i.centerID as centerId,
+                    COALESCE(i.centerID, 0) as centerId,
```

Comment:
Why are you adding this coalesce and using the key "0" instead of handling null appropriately? This is making the code in getSiteOptions confusing. 0 is not a valid centerid.

Reply:
@driusan For the Site filter to work in the data table, I use 0 as a flag for All Sites. Replacing NULL by 0 is convenient as React can filter successfully.

Reply:
Hmm, I think what may be better here (@driusan correct me if I'm wrong) is that if the value of `centerId` is NULL, then all centerIds currently in the psc table should be loaded into the issue row under the property `centerIds`. This would involve changing `int centerId` to `array centerIds` and loading only one value for most cases.

Reply:
I don't have any preference on if it's done the way Henri suggests or by leaving it null and deferring the loading of sites to later, but using "0" as a magic number is confusing.. if anything needs to be magic, it should be `null` (or `''` in places that can't be null)..